mono/utils/mono-threads-posix.c

```diff
@@ -103,6 +81,18 @@ win32_priority_to_posix_priority (MonoThreadPriority priority, int policy)
 void
 mono_threads_platform_register (MonoThreadInfo *info)
 {
+	gpointer thread_handle;
+
+	info->owned_mutexes = g_ptr_array_new ();
+	info->priority = MONO_THREAD_PRIORITY_NORMAL;
+
+	thread_handle = mono_w32handle_new (MONO_W32HANDLE_THREAD, NULL);
```

Comment:
I'm confused... this assigns to `thread_handle`, but it's never used anywhere.  Line 97 below has `info->handle = thread_handle_create ();`